blog/2021-09-15-soc2-things-i-learned-part-1.md

```diff
@@ -0,0 +1,142 @@
+In a lot of cases, many businesses won't even consider a B2B relationship if the other business is not SOC 2 compliant.**
+
+It's this exact reason why Temporal, as a startup-level business, decided to prioritize SOC 2 compliance early on.
+We knew that a SOC 2 report would serve as a confidence booster for our customers, and cut through a lot of the red tape that is otherwise associated with B2B relationships.
+We also hoped that the compliance process would help us organize ourselves internally to optimally reduce our exposure to risks and provide the best experience for our customers, which it did.
+
+## There are two types of SOC 2 reports
+
+Like the title says...
+
+### Type 1
+
+A Type 1 is a report that identifies compliance at a very specific point in time.
+For example if a Type 1 report is issued for March 31st 2021, then that means that on March 31st 2021, the organization appeared to be compliant with its control statements.
+When it comes to evidence gathering and analysis, it is easy to cherry pick in order to satisfy the control statement.
+For example, if the control statement states that "new employees agree to and sign an employee handbook", you really only need to provide one example of that happening to satisfy the audit.
+
+Because of this, there is usually a lot less friction in acquiring a SOC2 Type 1 report as a startup-level organization.
+The lower friction of a Type 1 means that you can use the audit as an exercise to identify your processes that can be captured as a control, and build up around the gaps.
+
+### Type 2
+
+A Type 2 report identifies compliance throughout a 6-12 month period of time.
+The organization can choose the period.
+What makes it more challenging is that the organization must be able to show that they are adhering to their control statements 100% of the time during that period.
+This means that for the same control that states "new employees agree to and sign an employee handbook", you must be able to show evidence that every single employee who joined the company in the period has signed the most recent version of the employee handbook.
+The auditors may not review each an every one, but their sampling is random.
+
+### How to choose
+
+My personal, high-level assessment here is that a startup-level organization should only really attempt a Type 2 of the gate, if the scope of the business is limited, there is minimal exposure to risk, and/or they have had enough time to establish and strictly adhere to internal processes that are industry standard best practices.
+
+At Temporal, we started with a Type 1 and used it as a way to build up to our Type 2.
+
+## Outsourcing only gets you so far
+
+**Ah, experience... the ultimate teacher.**
+
+At Temporal, we started looking at the SOC 2 compliance process in late 2020, but internally we did not have a lot of experience with it.
+For this reason we networked to find someone who did, to help us out.
```

Comment:
so like.. as an external reader i would be very surprised if any Soc2 discussion did not include Vanta. can we at least throw up a mention of why we did not explore vanta further?